enterprise/api_client/ecommerce.py

```diff
@@ -65,3 +65,41 @@ def get_course_final_price(self, mode, currency='$', enterprise_catalog_uuid=Non
         if price != mode['min_price']:
             return format_price(price, currency)
         return mode['original_price']
+
+    def create_manual_enrollment_orders(self, enrollments):
+        """
+        Calls ecommerce to create orders for the manual enrollments passed in.
+
+        `enrollments` should be a list of enrollments with the following format:
+        {
+            "lms_user_id": <int>,
+            "username": <str>,
+            "email": <str>,
+            "course_run_key": <str>
+        }
+
+        Since `student.CourseEnrollment` lives in LMS, we're just passing around dicts of the relevant information.
+        """
+        try:
+            order_repsonse = self.client.manual_course_enrollment_order.post(
+                {
+                    "enrollments": enrollments
+                }
+            )
+        except (SlumberBaseException, ConnectionError, Timeout) as exc:
+            LOGGER.exception(
+                "Failed to create order for manual enrollments for the following enrollments: %s. Reason: %s",
+                enrollments,
+                str(exc)
+            )
+        order_creations = order_repsonse["orders"]
+        successful_creations = [order for order in order_creations if order["status"] == "success"]
+        failed_creations = [order for order in order_creations if order["status"] == "failure"]
+        LOGGER.info(
+            "Successfully created orders for the following manual enrollments. %s",
+            successful_creations
+        )
+        LOGGER.error(
```

Comment:
Is logging here enough to raise awareness if orders haven't been created? I want to make sure we notice if there's failures.

Reply:
I'm not sure. We could probably check opsgenie and make a new alert if we can't find any existing generic error checks